server/models/testing/section.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const sectionSchema = new Schema({
+  name: { type: String, required: true, ref: 'Section' },
```

Comment:
You only need ref, when you know, that you will populate this field. Please read comment above & check documentation.